associacao-brasileira-de-normas-tecnicas-ufes-initials.csl

```diff
@@ -184,34 +214,34 @@
   <macro name="title">
     <choose>
       <if type="chapter bill paper-conference" match="any">
-        <text variable="title"/>
+        <text variable="title" text-case="title"/>
       </if>
       <else-if type="book thesis" match="any">
-        <text variable="title" font-style="normal" font-weight="bold"/>
+        <text variable="title" form="short" text-case="title" strip-periods="true" font-style="normal" font-weight="bold"/>
```

Comment:
@vict0rg, it's also rather uncommon to strip periods from titles with `strip-periods="true"`. E.g. a title like "Clustal W and Clustal X version 2.0" (https://doi.org/10.1093/bioinformatics/btm404) would incorrect render as "Clustal W and Clustal X version 20" with this setting.

Reply:
cc @alosaulo